src/MainWindow.cpp

```diff
@@ -4562,6 +4578,54 @@ void MainWindow::setupViews()
     showHideViewColumnsForAccountType(currentAccountType);
 }
 
+bool MainWindow::getShowNoteThumbnails() const
+{
+    ApplicationSettings appSettings(*m_pAccount, QUENTIER_UI_SETTINGS);
+    appSettings.beginGroup(LOOK_AND_FEEL_SETTINGS_GROUP_NAME);
+    QVariant showThumbnails = appSettings.value(SHOW_NOTE_THUMBNAILS_SETTINGS_KEY,
+        QVariant::fromValue(DEFAULT_SHOW_NOTE_THUMBNAILS));
+    appSettings.endGroup();
+
+    return showThumbnails.toBool();
+}
+
+QSet<QString> MainWindow::getHideNoteThumbnailsFor() const
+{
+   ApplicationSettings appSettings(*m_pAccount, QUENTIER_UI_SETTINGS);
```

Comment:
Since you are going to push one more commit anyway, could you please adjust the indentation here? It's one column off to the left.